frontend/tests/controllers-test/profileCtrl.test.js

```diff
@@ -70,7 +71,7 @@ describe('Unit tests for profile controller', function () {
             for (var i = 0; i < successResponse.length; i++) {
                 if (successResponse[i] === "" || successResponse[i] === undefined || successResponse[i] === null) {
                     successResponse[i] = "-";
-                    expect(vm.countLeft).toEqual(vm.countLeft + 1);
+                    expect(vm.countLeft).toEqual(vm.countLeft);
```

Comment:
What does this test anyway?

Reply:
This and other instance of countLeft is basically to check if the value we get for any field except the url fields is null or empty if it is then we add a "-" to it for display purposes, i replaced that with emtpy as i have put other way of display this with doesn't affect our dialog box values